website/blog/2023-01-18-release-0.11.md

```diff
@@ -97,25 +100,24 @@ The experience of configuring a registry is getting simplified for the most popu
 
 <ReactPlayer playing controls url='https://user-images.githubusercontent.com/6422176/214332937-eb1d9050-0d32-4bc4-8393-49b4583b1390.mov' />
 
-### UI/UX Improvements 
+### UI/UX Improvements
 
 #### View Pods logs [#1122](https://github.com/containers/podman-desktop/pull/1122)
 
-The pods details view provides the ability to view the logs of each containers that may be running in a pod. Moreover, as it can also be hard to scan and identify which container is producing which output, we appended the container name at the beginning of each output and differentiate each container with a different color. 
+The pods details view provides the ability to view the logs of each containers that may be running in a pod. Moreover, as it can also be hard to scan and identify which container is producing which output, we appended the container name at the beginning of each output and differentiate each container with a different color.
```

Comment:
```suggestion
The pods details view provides the ability to view the logs of each containers that might be running in a pod. Moreover, as it can also be hard to scan and identify which container is producing which output, we appended the container name at the beginning of each output and differentiate each container with a different color.
```